versioned_docs/version-v2.2.0/developers/components/WebSocketHandler.jsx

```
@@ -0,0 +1,52 @@
+    return () => {
+      if (!socketRef.current) return
+      socketRef.current.removeEventListener('message', handleReceivedMessage)
+    }
+  }, [socketRef.current])
+
+  return (
+    <>
+      {multipleMessages ? (
+        <div>
+          {messages.map((message) => (
+            <p>{message}</p>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)